src/unix/unix.c

```diff
@@ -472,7 +499,7 @@ process init_unix(kernel_heaps kh, tuple root, filesystem fs)
 
     u_heap = uh;
     uh->kh = *kh;
-    uh->processes = create_id_heap(h, h, 1, 65535, 1, false);
+    uh->processes = locking_heap_wrapper(h, (heap)create_id_heap(h, h, 1, 65535, 1, false)); // XXX shared pid/tid space may need to be much much larger
```

Comment:
The maximum number of Linux threads can scale with memory into the hundreds of thousands. I don't know that we'd ever run applications that would have that many though, so we probably don't need to adjust it. I'll remove the comment, because it's very unlikely to be a problem.